### ISSUE TITLE
Testi in inglese

### ISSUE THREAD

**Issue by NarakuITA** [tesivo/VTMB_italian_translation_for_Unofficial_Patch#32]
Seconda risposta:
![Vampire The Masquerade - Bloodlines Screenshot 2021 06 18 - 15 15 21 43](https://user-images.githubusercontent.com/38878793/122655192-f866af00-d150-11eb-83e6-f5273e922916.png)
![Vampire The Masquerade - Bloodlines Screenshot 2021 06 18 - 15 41 34 73](https://user-images.githubusercontent.com/38878793/122655197-04527100-d151-11eb-9c58-e501be388e5e.png)
Terza risposta:
![Vampire The Masquerade - Bloodlines Screenshot 2021 06 18 - 23 01 57 73](https://user-images.githubusercontent.com/38878793/122655205-103e3300-d151-11eb-8a6b-2578b8c3433c.png)
![Vampire The Masquerade - Bloodlines Screenshot 2021 06 19 - 22 12 25 31](https://user-images.githubusercontent.com/38878793/122655213-16341400-d151-11eb-826a-887845999024.png)

**Comment by NarakuITA** [comment]
> Tutto ciò che hai segnalato tranne la seconda immagine, è ora sistemato.
> 
> Sì, sicuramente ci sono ancora parecchi testi da tradurre, diciamo che dove manca un discorso intero devo ancora metterci mano. Voi segnalatemi le frasi sparse e dove sembra un discorso intero mancante(senza condividere tutte le frasi incriminate).
> 
> Grazie ;)

Ok, proverò a non mandarti troppe immagini, ma solo i testi! Cmq ben tornato!

**Comment by tesivo** [comment]
Ah a me non danno alcun fastidio le immagini ;) era per semplificarti la vita :D

**Comment by NarakuITA** [comment]
Questi sono dei dialoghi, generali, dei vari PG che sono in giro per la città:

- Man: Of all the places to have car problems-
- Male: Good lock pinding a bidet around here.
- Male: Oh, what is that on my loaper?
- Male: I'm going to need to bathe when I get home.
- Male: The taxpayers are certainly not getting their money's worth here.
- Male: I'm going to need a shower.
- Male: Shh, mayhem, bro.
- Male: Ah, the hell kinda music is this?
- Male: I don't think so, duddy. 
- Male: Yeah, right back at ya.
- Male: Hey, big spender, yeah...
- Male: Tja.
- Male: It's a nice night out, huh?
- Male: Don't touch me, huh?
- Female: I'll scream.
- Female: Buy a watch.
- Female: Hey baby, need company?
- Female: Eh, not interested. 
- Female: What ever.
- Female: What?!
- Whisper: Maggots love you.
- Whisper: Trust me.

**Comment by tesivo** [comment]
Che traduzione adottereste per "There that crab nipple 'n this'n's got crab nipple - whole lotta crab nipple."?

**Comment by NarakuITA** [comment]
> Che traduzione adottereste per "There that crab nipple 'n this'n's got crab nipple - whole lotta crab nipple."?

Ho chiesto aiuto nel forum Traduzioni. Vediamo gli altri cosa propongono. E ti farò sapere al più presto.

**Comment by tesivo** [comment]
Intanto ti dico che ho trovato tutto il gruppo di frasi che mi hai segnalato e pure altre ;) (ed è in arrivo una sorpresa, forse la svelo nel we)

**Comment by NarakuITA** [comment]
Una sorpresa? bene! XD

**Comment by NarakuITA** [comment]
> Che traduzione adottereste per "There that crab nipple 'n this'n's got crab nipple - whole lotta crab nipple."?

Senti questa frase in che contesto viene detta? E' un insulto di un boss vampiro?

**Comment by Root78** [comment]
Da quello che mi ricordo è un altro dei dialoghi che viene detto quando ti avvicini, come "buy a watch" e gli altri. Quindi non c'è un vero contesto, quindi tradurrei la frase così com'è.

**Comment by NarakuITA** [comment]
> Da quello che mi ricordo è un altro dei dialoghi che viene detto quando ti avvicini, come "buy a watch" e gli altri. Quindi non c'è un vero contesto, quindi tradurrei la frase così com'è.

Il contesto praticamente è il seguente: Sono due barboni, ubriachi che fanno sesso tirandosi i capezzali!  Ora Per quanto riguarda il forum traduzioni, nada, de nada. Allora ho chiesto a un'amica, di aiutarmi, molto brava con l'inglese, quasi madre lingua. Vediamo che mi dirà.

**Comment by NarakuITA** [comment]
Questa è la traduzione migliore che è possibile fare: 

Ecco qui una bella tirata di capezzolo e anche a questo qui una bella tirata di capezzolo-vuoi un sacco di tirate di capezzolo?

**Comment by tesivo** [comment]
Ottimo! La carico subito. Grazie a voi e grazie all'amica ;)

**Comment by crazyLillo** [comment]
salve, con umiltà vi propongo questa 

There that crab nipple 'n this'n's got crab nipple - whole lotta crab nipple.

Ecco una strizzata a quel capezzolo e anche a questo una bella strizzata - un sacco di strizzate ai capezzoli

**Comment by tesivo** [comment]
Non mi sembra affatto male quest'ultima traduzione. 

Cosa ne pensi @NarakuITA ? Il senso è quello, ma mi sembra scorrere meglio.

**Comment by NarakuITA** [comment]
> Non mi sembra affatto male quest'ultima traduzione.
> 
> Cosa ne pensi @NarakuITA ? Il senso è quello, ma mi sembra scorrere meglio.

Non, male, ma visto che abbiamo crazyLillo, che ha vissuto a Los Angeles e conosce bene lo slang usato, forse lui può aiutarci meglio! @crazyLillo che ne dici?

**Comment by tesivo** [comment]
Ottimo, nell'augurarvi un buon anno, carico questa modifica e mi scuso in anticipo se le mie traduzioni sono da inesperto della lingua e spesso coadiuvate da traduttori come DeeplTranslator :D

Buon anno ragazzi! ;)

**Comment by crazyLillo** [comment]
Bhe! in L.A ho fatto un po di vita da strada alla bukowski ( fine anni
80...) non sono stato proprio uno stinco di santo nella vita... conosco un
po di slang, ma è come dire conoscere un po di napoletano stretto o

bergamasco, le lingue sono così diverse e complesse...
lo slang di Los Angeles è un dialetto americano, quando la lingua americana
tende ad accorcciare la lingua classica inglese, lo slang la accorcia
ancora di più, poi se ci aggiungiamo  i termini che cambiano di zona in
zona

della california....

molte frasi di tutte le traduzioni fatte finora mi lasciano molto perplesso
e quasi disgustato (non posso farci nulla è piu forte di me...).

bhe! tagliando corto, preparatevi alla rivoluzione

non solo mi accingo a fare una revisione di questi testi maledetti, ma sto
lavorando per un possibile doppiaggio (solo del tutorial).
non parlo a casaccio, ho gia effettuato le prove e ho il doppiaggio della
prima frase pronto all'uso ( e scusate se è poco, anche con il labiale
sincronizzato!)
ho dato anche una nuova traduzione (quella vera) a quella prima frase.
l'unica cosa e che mi serve un microfono piu performante di questa piccola
cuffietta con il microfonino
il file audio che ne esce fuori e troppo sporco per i miei gusti, anche
elaborandolo con software, non migliora di molto.
morale, mi serve un microfono vero per fare maledettamente sul serio.

ad ogni modo, questa è la mia analisi sulla prima frase del tutorial, a voi
i commenti

originale

[laughing at something no one else thinks is funny]What a scene, man!
Hoo-wee! Then they just plop ya out here like a naked baby in the woods.
[chuckle]How 'bout that? Ah... look, kiddo, this is probably a lot for you
to

take in, so uh, why don't you let me show you the ropes. Whaddya say?

traduzione attuale

[ride di qualcosa che nessun altro pensa sia divertente]Che spettacolo,
amico! Ti hanno buttato qui proprio come un bambino nudo nella foresta. Non
e' vero? Senti, ragazzo, probabilmente e' troppo per te, quindi perche'

non lasci che ti spieghi come funziona. Cosa ne dici?


la MIA traduzione (con un possibile audio di doppiaggio sincronizzato al
labiale di Jack)

[ride di qualcosa che nessun altro pensa sia divertente] Haa ha ha haa ha
Che scena amico! Hoo-wee! ha ha ha Così ti hanno scaricato quì fuori come
un bimbo nudo nel bosco. [risatina beffarda] he he Che ne pensi?

Ahhh....guarda, ragazzo, questo è probabilmente troppo per te da assorbire,
quindi eeh, perchè non lasci che ti mostri le cose. Che ne dici?


BUON ANNO!




Il giorno sab 1 gen 2022 alle ore 00:17 Lorenzo Rubagotti <
***@***.***> ha scritto:

**Comment by tesivo** [comment]
Riprendiamo il progetto: la traduzione è ora aggiornata alla Unofficial Patch 11.5 (con anche la Plus Patch opzionale).

Le frasi sparse segnalate in questo issue, inclusa quella dei due barboni, risultano già integrate nella traduzione da tempo. Chiudiamo questo issue: se emergono altre frasi non tradotte tra i dialoghi casuali dei passanti, apriteci pure una nuova segnalazione.
